File: examples/ip/lookup/go/lookupIpInfo.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/ip"
)

// Lookup the geolocation information for an IP address
func LookupIpInfo() {
	ipService := ip.NewIpService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := ipService.Lookup(&ip.LookupRequest{
		Ip: "93.148.214.31",
	})
	fmt.Println(rsp, err)
}

File: examples/ip/lookup/node/lookupIpInfo.js
import * as ip from "m3o/ip";

// Lookup the geolocation information for an IP address
async function LookupIpInfo() {
  let ipService = new ip.IpService(process.env.MICRO_API_TOKEN);
  let rsp = await ipService.lookup({
    ip: "93.148.214.31",
  });
  console.log(rsp);
}

await LookupIpInfo();
